back/src/messages/output-processor.ts

```diff
@@ -37,9 +41,20 @@ export const processOuputMessage = (socketInfo: SocketInfo, action: Action) => {
       handleAppendText(socketInfo, action.payload);
     case OutputMessageTypes.REPLACE_FULL_TEXT:
       handleReplaceFullText(socketInfo, action.payload);
+    case OutputMessageTypes.STUDENT_SEND_FULL_CONTENT:
+      handleStudentSendContent(socketInfo, action.payload);
   }
 };
 
+const handleStudentSendContent = (socketInfo: SocketInfo, room: string) => {
```

Comment:
I thin we can remove room from there, and obtain it via socketInfo, then we get the content or something like that